itou/templates/dashboard/api_token.html

```diff
@@ -0,0 +1,56 @@
+{% extends "layout/content_small.html" %}
+{% load bootstrap4 %}
+
+{% block title %}Mes notifications{{ block.super }}{% endblock %}
```

Comment:
Mes notifications ?
Il manque l’espace avant le tiret.
```suggestion
{% block title %}Mon token d’API {{ block.super }}{% endblock %}
```

Reply:
C/C d'un autre template pour le POC en oubliant de modifier cette ligne, 
merci